common-service/src/main/java/org/sunbird/util/AuditLogUtil.java

```diff
@@ -0,0 +1,26 @@
+package org.sunbird.util;
```

Comment:
Remove this file for now. Later include if needed when adding audit log utility methods.